### DISCUSSION TITLE
RO Crates

### DISCUSSION THREAD

**Discussion by johanneswerner** [nfdi4plants/ARC-specfication#47] [Comments and Suggestions]
As discussed in the speaker's meeting, this topic is rather important and thus, I advise to not list it within the ARC specifications as appendix but rather would outsource it to a new document (and fill it with more details).

**Comment by muehlhaus** [comment]
The link to ROCs is very import and ARCs are intended to be first class ROCs. However, this is no hard requirement. Therefore, we decided to place it into the appendix.
